packages/dds/shared-object-base/src/sharedObject.ts

```diff
@@ -572,6 +582,30 @@ export abstract class SharedObjectCore<
 		this.emitInternal("op", message, local, this);
 	}
 
+	/**
+	 * Process messages for this shared object. The messages here are contiguous messages for this object in a batch.
+	 * @param message - The core message with common properties for all the messages.
+	 * @param messageContents - The contents, local metadata and clientSequenceNumbers of the messages.
+	 * @param local - true if the messages were originally generated by the client receiving it.
+	 */
+	private processMessages(
+		message: ISequencedRuntimeMessageCore,
```

Comment:
One thing to consider - instead of passing 3 args, maybe we lump them into single object and give it a type.
Maybe then we do not need separate ISequencedRuntimeMessageCore interface, not sure.

Reply:
This might make future changes easier. For example, maybe we will want to pass localOpMetadata in the future (I think DDSs need it, no?)

Reply:
Changed it.